tests/integration-tests/tests/runtime_bake/test_runtime_bake/test_runtime_bake/verify_cinc_download.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -x
 
+# get cloudformation log if it doesn't exist
+if [ ! -d "/var/log/cloud-init-output.log" ]; then
```

Comment:
`-d` tests if something is a directory. i don't think that's what you want here.